### ISSUE TITLE
Configuring set of validators for Mir (membership)

### ISSUE THREAD

**Issue by adlrocha** [consensus-shipyard/lotus#5]
Even if we reconfiguration is not supported in Mir in the first release, we still need to provide the protocol with the set of validators for its operation. Whatever scheme we choose to pass the validator set to Mir, it should be flexible enough to support adding and removing validators easily.

Initially, we are considering the persistence of the validator set in an on-chain actor. Mir will access the state of the actor to determine the list of active validators (similar to how Mir works currently in subnets in Eudico). This is subject to change and requires further design and discussion.

**Comment by jsoares** [comment]
@adlrocha @matejpavlovic what's the plan here in view of spacenet?

**Comment by adlrocha** [comment]
For M1 and the Mir rootnet we are following the same plan we designed for buildernet:
- Initially we will only support static membership (although reconfiguration is already implemented also in the Lotus integration, so if we want to test with reconfiguration we can, however, I would suggest narrowing the testing surface initially for the November launch).
- Membership is currently handled by a config file in every validator (actually, this issue can be moved to the "review" state because it was implemented as part of https://github.com/filecoin-project/lotus/pull/9492).
- For M3, the idea is to move membership on-chain as part of IPC.
